### PR TITLE
Vertically align close and kebab icons in notification drawer

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://github.com/openshift/origin-web-console/issues/2218

screen shot 2017-10-06 at 2 05 27 pm

### CODE REVIEW

File: app/styles/_notifications.less [hunk 1]
```diff
@@ -59,10 +59,14 @@ notification-drawer-wrapper {
...
     padding: 15px;
-    .pficon-close {
-      color: @color-pf-black-400;
-      &:hover,&:focus,&:active {
-        color: @color-pf-black-700;
+    .drawer-pf-notification-close {
+      margin-right: (@grid-gutter-width/(-4)); // vertically align with dropdown-kebab-pf
+      .pficon-close {
+        color: @color-pf-black-400;
+        &:hover,&:focus,&:active {
```
Comment (rhamilto) [inline app/styles/_notifications.less]: Nit:  alpha and maybe put each selector on a separate line?

**Comment by sg00dwin** [comment]
@spadgett @rhamilto pr updated so ` .pf-close` color  matches `.dropdown-kebab-pf` 
https://github.com/patternfly/patternfly/blob/master/src/less/dropdowns.less#L169

screen shot 2017-10-06 at 4 09 19 pm

**Comment by benjaminapetersen** [comment]
[test]

Just need to see if Travis can start Chrome on other branches...

**Comment by benjaminapetersen** [comment]
Aha. e2e tests not running in Travis here.

**Comment by sg00dwin** [comment]
@spadgett updated pr to use `<button>` instead of `<a>`

File: app/styles/_notifications.less [hunk 2]
```diff
@@ -59,10 +59,16 @@ notification-drawer-wrapper {
...
     padding: 15px;
-    .pficon-close {
-      color: @color-pf-black-400;
-      &:hover,&:focus,&:active {
-        color: @color-pf-black-700;
+    .drawer-pf-notification-close {
+      margin-right: (@grid-gutter-width/(-4)); // vertically align with dropdown-kebab-pf
+      .pficon-close {
+        color: @gray-darker; // match dropdown-kebab-pf color
+        &:active,
```
Comment (rhamilto) [inline app/styles/_notifications.less]: This rule is no longer needed now that the close is a button.

File: app/views/directives/notifications/notification-body.html [hunk 3]
```diff
@@ -2,14 +2,13 @@
...
   ng-click="$ctrl.customScope.markRead(notification)">
-  <a
-    class="pull-right"
-    href=""
+  <button
+    class="btn btn-link pull-right drawer-pf-notification-close"
+    type="button"
     ng-if="!notification.actions.length"
     ng-click="$ctrl.customScope.clear(notification, $index, notificationGroup)">
-    <span class="sr-only">Clear notification</span>
```
Comment (spadgett) [inline app/views/directives/notifications/notification-body.html]: Looks like you lost the sr-only text

**Comment by spadgett** [comment]
[merge]

**Pull request merged**